client/blocks/conversations/empty.jsx

```diff
@@ -34,7 +34,7 @@ class ConversationsEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read/search"
 				>
-					{ this.props.translate( 'Find Posts to Follow' ) }
+					{ this.props.translate( 'Find posts to follow' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.